DRAFT_nature_index.Rmd

```diff
@@ -0,0 +1,382 @@
+```{r}
+Diagnostics <- TRUE # Yes
+TestRun <- TRUE # Yes
+```
+The difference for the latter is that in a test run, we only run NA imputation and bootstrap for 10 iterations instead of 1000, which makes the runtime of the workflow a lot more "practical" (but 1000 or even more iterations should be run for the final results).
+
+
+### Scaled indicator values
+Before beginning index calculation, there are a few more conditional variables that need to be set, namely the Norwegian name of the ecosystem and the lists of indicators to use and indicators to drop. 
+```{r}
+ecosystem_use <- EC_ecosystem[2]
+indicators_use <- NULL
+
+dropInd <- selectDropIndicators(DropIndMode = DropIndMode,
+                                OutputType = OutputType,
+                                ecosystem = ifelse(OutputType %in% c("NatureIndex", "EcologicalCondition"), ecosystem_use, NULL),
+                                customList = DropIndList)
+```
+Note that we set `indicators_use` to `NULL` because we are working with a pre-defined set of indicators (i.e. the set linked to NI for wetlands).
+
+We also need to set variables "theme" and "funArguments" to "none" and `NULL`, respectivel (these are used for controlling workflow elements linked to thematic indices):
+```{r}
+theme <- "none"
+funArguments <- NULL
+```
+
+With this, we are ready to move ahead with the calculation of our wetlands NI product. For this, we use `NIflex::calculateCustomNI()`, a wrapper function that downloads the data, runs dataset diagnostics (if requested), imputes NA values (if requested), and calculates the NI product using functions from the NIcalc package: 
+```{r, results = FALSE}
+CustomNI <- calculateCustomNI(ecosystem = ecosystem_use,
+                              indicators = indicators_use,
+                              theme = theme,
+                              dropInd = dropInd,
+                              username = NIdb_username,
+                              password = NIdb_password,
+                              KeyIndicators = KeyIndicators,
+                              KeyWeight = KeyWeight,
+                              AreaWeights = AreaWeights,
+                              TrophicWeights = TrophicWeights,
+                              NAImputation = NAImputation,
+                              years = years,
+                              OutputType = OutputType,
+                              funArguments = funArguments,
+                              Diagnostics = Diagnostics,
+                              TestRun = TestRun,
+                              norwegianNames = norwegianNames,
+                              saveSteps = FALSE)
+```
+
+After calculating the index values, we can extract summary metrics for a 
+specified year and match them to geospatial data for further analysis and/or 
+inclusion in the Ecosystem Condition framework: 
+
+```{r}
+CustomNI_map <- geomapNI(Index = CustomNI$CustomIndex, 
```

Comment:
I get stuck here due to missing permissions on the folder (awaiting update from datahjelp) 
Quitting from lines 283-288 [unnamed-chunk-17] (DRAFT-Nature-Index-product.Rmd)
Error:
! Cannot open "/data/P-Prosjekter/41201611_naturindeks_2021_2023_vitenskapelig_ansvar/Shapefiles/NI_ecosystems/NI_ecosystem_Terrestrial"; The file doesn't seem to exist.